File: CRM/src/app/shared/eHospital_DongNai_A_NSTL/NS_RP_LSTREPORT.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_RP_LSTREPORT } from './NS_RP_LSTREPORT.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_RP_LSTREPORTService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'REPORTCODE', 'REPORTTITLE', 'REPORTFILENAME', 'PACKAGENAME', 'PROCEDURENAME', 'PACKAGENAME_ORIGIN_DATA', 'TableMapping', 'REPORTTYPE', 'Sort'];

    List: NS_RP_LSTREPORT[] | undefined;
    ListFilter: NS_RP_LSTREPORT[] | undefined;
    FormData!: NS_RP_LSTREPORT;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_RP_LSTREPORT";
    }
}

File: Service_eHospital_DongNai_A_NSTL/Implement/NS_RP_LSTREPORTService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_RP_LSTREPORTService : BaseService<NS_RP_LSTREPORT, INS_RP_LSTREPORTRepository>
    , INS_RP_LSTREPORTService
    {
    private readonly INS_RP_LSTREPORTRepository _NS_RP_LSTREPORTRepository;
    public NS_RP_LSTREPORTService(INS_RP_LSTREPORTRepository NS_RP_LSTREPORTRepository) : base(NS_RP_LSTREPORTRepository)
    {
    _NS_RP_LSTREPORTRepository = NS_RP_LSTREPORTRepository;
    }
    }
    }

